src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/AttributeGroup/Blocks/form.html.twig

```diff
@@ -0,0 +1,53 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+
+{% form_theme attributeGroupForm '@PrestaShop/Admin/TwigTemplateForm/prestashop_ui_kit.html.twig' %}
+
+{% block attribute_group_form_block %}
+  {{ form_start(attributeGroupForm) }}
+
+  <div class="card">
+    <h3 class="card-header">
+      {{ 'Attribute Group'|trans({}, 'Admin.Catalog.Attribute') }}
```

Comment:
`{{ 'Attribute'|trans({}, 'Admin.Catalog.Attribute') }}`

Reply:
I think this new wording is more adapted, to understand the difference between Attribute group and its contained Attributes

Reply:
So to clarify it should stay as Attribute group right?

Reply:
@jolelievre @JevgenijVisockij @l-delin It has always been `Attribute` and `Attribute value` and I think it should stay like that.

It's called `Attribute group` only in the code and in the database.